interfaces/Portalicious/src/app/domains/payment/payment.api.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable, Signal } from '@angular/core';
+import { DomainApiService } from '~/domains/domain-api.service';
+import { Payment } from '~/domains/payment/payment.model';
+
+const BASE_ENDPOINT = (paymentId: Signal<number>) => [
+  'programs',
+  paymentId,
+  'payments',
+];
```

Comment:
```suggestion
const BASE_ENDPOINT = (projectId: Signal<number>) => [
  'programs',
  projectId,
  'payments',
];
```

Right?

Reply:
Nee, all of the payments endpoints are a subset of the project controller. This was the thing preventing me from separate payments from projects in the previous iteration. You can, for example, get all of the payments for a specific project and for that you do not need a paymentId but you always need a projectId (because all payments are tied to a project).